src/components/slider/GradientSlider.tsx

```diff
@@ -83,9 +83,9 @@ const GradientSlider = (props: Props) => {
     ...others
   } = props;
 
-  const [initialColor] = useState(Colors.getHSL(propsColors));
+  const initialColor = useRef(Colors.getHSL(propsColors));
   const [color, setColor] = useState(Colors.getHSL(propsColors));
-
+  
```

Comment:
Remove extra space

Reply:
This is not extra, I separated the consts and the useEffect